go.mod

```diff
@@ -3,20 +3,20 @@ module maistra.io/api
 go 1.15
 
 require (
-	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
+	github.com/ghodss/yaml v1.0.0
```

Comment:
Where did these changes come from?  They don't seem directly related to the change.

Also, are we running `go mod tidy` when we update go.mod?

Reply:
1.0.1 came from initial commit. Now I'm lowering the versions so that they match whatever is in maistra/istio so that changes there are the minimum possible. This change is a separate commit, btw.

Yes, we run go mod tidy, even as part of make gen-check.